### DISCUSSION TITLE
How Can I plot all the dates in a list ( vertically) on candelstick chart?

### DISCUSSION THREAD

**Discussion by qn222ad** [highfestiva/finplot#176] [Q&A] [answered]
In finplot candestick chart,  I had a list of dates, I want to plot all of these date on chart in vertical lines, how can I do it?
Also in the candlestick chart, as I move the mouse I saw the date show up, how can I onclick and save that date?
Thanks
Nga

**Comment by highfestiva** [comment, marked as answer]
You could use `set_time_inspector()` using `when='hover'` to update a text showing the date, see `examples/snp500.py` for more info. If you rather want the click, use `when='click'` instead.

**Reply by qn222ad** [reply]
Thank you very much, how about plot all vertical lines in a list of dates? Does this had support?

**Reply by highfestiva** [reply]
Check out the bitcoin long term example.

**Reply by qn222ad** [reply]
Thanks:):)

**Reply by qn222ad** [reply]
Both of the codes run good now :):). Can I ask, if I plotted the lines, how I can remove any line on click? or double click?

**Reply by qn222ad** [reply]
Can you give me a help. I run the code on two different laptops. The first one the dates was plot fine, however, the second one is not plot as desire. How should I fix them?
Thanks:)
![photo_2021-05-06_17-40-41](https://user-images.githubusercontent.com/77920707/117326740-3efa9580-ae92-11eb-9f53-febfb8add629.jpg)

![2021-05-06 17_35_54-Window](https://user-images.githubusercontent.com/77920707/117326705-36a25a80-ae92-11eb-87c0-634ec4b0b3f7.png)


fplt.show()

**Reply by highfestiva** [reply]
Loot at how it's done in the examples.

**Reply by qn222ad** [reply]
I tested again by creating a new project, it run normally, then the problem might be related to the other library I installed in the old project....
